overrides/android-override.json

```
@@ -358,6 +358,27 @@
                         ]
                     }
                 },
+                "siteSpecificFixes": {
+                    "state": "disabled",
+                    "settings": {
+                        "javascriptConfig": {
```

As mentioned elsewhere, I don't see the rationale for this javascriptConfig nesting. All other subfeatures (except password import) don't have this nesting.

For password import the use-case was different we translate `javascriptConfig` into a top level feature in a totally different context.

I don't think we want another layer of nesting for all JS features going forward.

---

Removed! And types updated